package.json

```diff
@@ -19,7 +19,8 @@
   },
   "configSchema": "nut-config-schema.coffee",
   "dependencies": {
-    "node-nut": "0.1.0"
+    "node-nut": "1.0.3",
+    "pimatic": "^0.9.50"
```

Comment:
Pimatic dependency is defined in "peerDependencies" actually. Could you test it removing this line?